core/src/com/unciv/models/ruleset/Difficulty.kt

```diff
@@ -23,11 +25,23 @@ class Difficulty: INamed {
     var aiBuildingMaintenanceModifier:Float = 1f
     var aiUnitMaintenanceModifier = 1f
     var aiFreeTechs = ArrayList<String>()
+    @Deprecated("Use aiMajorCivStartingUnits instead")
```

Comment:
Please add from what version this is deprecated so that later we'll know how long it was and whether it's time to delete it or not :)

Reply:
added